src/app/search/search.component.ts

```diff
@@ -410,6 +421,41 @@ export class SearchComponent implements OnInit, OnDestroy {
       .catch((error) => console.log(error));
   }
 
+  /**
+   * Updates the results table when there is no search term
+   * Queries for 100 tools and 100 workflows
+   *
+   * @param {string} value the elastic search query
+   * @memberof SearchComponent
+   */
+  updateResultsTableSeparately(value: string) {
+    let toolHits;
+    let workflowHits;
+    const numOfResults = 201;
+    // Combine array of tools and workflows to filter into results table
+    ELASTIC_SEARCH_CLIENT.search({
+      index: 'entries',
+      type: 'tools',
+      body: value,
+    })
+      .then((hits) => {
+        toolHits = hits.hits.hits;
+      })
+      .catch((error) => console.log(error));
+    ELASTIC_SEARCH_CLIENT.search({
+      index: 'entries',
+      type: 'workflows',
+      body: value,
+    })
+      .then((hits) => {
+        workflowHits = hits.hits.hits;
+        this.hits = toolHits.concat(workflowHits);
```

Comment:
there is no guarantee that toolHits will return before workflowHits, need to join them somehow